routes/auth.js

```diff
@@ -11,14 +12,18 @@ module.exports = (app, passport) => {
     "/login",
     passport.authenticate("local"),
     async (req, res, next) => {
-      console.log("making it here?");
-      const user = req.user;
-      console.log(user, "req.user TESTTTTTTTTT");
-      if (!user) {
-        // we didnt get a user back from passport authenticate
-        res.status(401).send({ message: "Incorrect username or password" });
+      try {
+        console.log("making it here?");
+        const user = req.user;
+        console.log(user, "req.user TESTTTTTTTTT");
+        if (!user) {
+          // we didnt get a user back from passport authenticate
+          throw new ExpressErrorHandler(401, "Incorrect username or password");
```

Comment:
I believe if there is an error like this, you should throw an instance of Error. You currently have a really good architecture when handling errors, which is that you throw an error and then let `next()` take it to your central error handler. 

On top this error was not in the try-catch block so I moved it into the block.